addons/point_of_sale/models/pos_order.py

```diff
@@ -476,7 +476,7 @@ def create_picking(self):
             if picking_type:
                 message = _("This transfer has been created from the point of sale session: <a href=# data-oe-model=pos.order data-oe-id=%d>%s</a>") % (order.id, order.name)
                 picking_vals = {
-                    'origin': order.name,
+                    'origin': '%s - %s' % (self.session_id.name,order.name),
```

Comment:
`self.session_id` could crash:
```suggestion
                    'origin': '%s - %s' % (order.session_id.name, order.name),
```